eng/pipelines/templates/Build_and_UnitTest.yml

```diff
@@ -113,7 +113,7 @@ steps:
   inputs:
     solution: "src\\NuGet.Clients\\NuGet.CommandLine\\NuGet.CommandLine.csproj"
     configuration: "$(BuildConfiguration)"
-    msbuildArguments: "/t:ILMergeNuGetExe /p:ExpectedLocalizedArtifactCount=$(LocalizedLanguageCount) /bl:$(Build.StagingDirectory)\\binlog\\04.ILMergeNuGetExe.binlog"
+    msbuildArguments: "/t:Build,ILMergeNuGetExe /bl:$(Build.StagingDirectory)\\binlog\\04.ILMergeNuGetExe.binlog"
```

Comment:
Wouldn't this build again?

Reply:
Yes, I need to build first to get binaries to be localized

Later, when satellite assemblies are built, I need to embed those satellite resource assemblies into nuget.exe; This is why I double-build nuget.exe

Reply:
I'm still missing why you need to double build nuget.exe. 

Once the satellite assemblies are build, you should only ilmerge correct?

Reply:
> Once the satellite assemblies are build, you should only ilmerge correct?
No. Resource assemblies are now included in nuget.exe as Embedded Resources.
- First build creates the nuget.exe binary
- Second build creates the nuget.exe binary with the satellite assemblies embedded, not ilMerged

![image](https://user-images.githubusercontent.com/1192347/187570942-b005f79e-b2bd-4f79-8e1d-38c6d15c09a3.png)


The final ILMerge is for including code dependent assemblies into nuget.exe.

Reply:
Thanks for that explanation. 

So you are basically embedding so that in the assembly resolve event you can look up a dependent assembly as a resource, correct?

Reply:
Correct.